Libplanet.Tools/Mpt.cs

```diff
@@ -157,6 +129,13 @@ public class Mpt
             string uri,
             [FromService] IConfigurationService<ToolConfiguration> configurationService)
         {
+            if (Uri.IsWellFormedUriString(alias, UriKind.Absolute))
+            {
+                throw new CommandExitedException(
+                    "The alias should not be well-formed URI. Please try to use other alias name.",
```

Comment:
```suggestion
                    "The alias should not look like a URI to prevent it being ambiguous." +
                    "Please try to use a different alias.",
```

Reply:
It was amended at 459c16d28fd82fc2bb585a2c8419c47ee59e2015.